README.md

```diff
@@ -32,7 +32,11 @@ at their own discretion.
     help projects adopt and iterate on policies and processes that achieve
     these goals and eventually allow them to graduate the incubation phase.
 
-      * Mikeal Rogers (@mikeal)
+      * Mikeal Rogers (@mikeal) Currently assigned to Express.
+      * James Snell (@jasnell) Currently assigned to Express.
+      * Rod Vagg (@rvagg) Currently assigned to libuv.
+      * Alexis Campailla (@orangemocha) Currently assigned to libuv.
+
```

Comment:
Yeah your alias is in the issue body as a comment, which does ping the user and pull them into the issue as a subscriber, but in a repo `README.md` file it won't do that. Hope this clears it up for you. :)